pkg/controller/remotemachineset/remotemachineset_controller.go

```diff
@@ -128,6 +129,10 @@ func (r *ReconcileRemoteMachineSet) Reconcile(request reconcile.Request) (reconc
 		log.WithError(err).Error("error looking up cluster deployment")
 		return reconcile.Result{}, err
 	}
+	if cd.Annotations[constants.SyncsetPauseAnnotation] == "true" {
+		log.Warn(constants.SyncsetPauseAnnotation, " is present, hence syncing to cluster is disabled")
```

Comment:
Usually would see this as log.Warnf("%s is present, hence ...", constants.SyncsetPauseAnnotation) but no idea if either form is better or why.

Reply:
Just a comment, not a necessary change.